src/rusoto/auth.rs

```diff
@@ -12,6 +12,10 @@ pub enum AwsAuthentication {
         access_key_id: String,
         secret_access_key: String,
     },
+    File {
+        credentials_file: String,
+        profile: String,
```

Comment:
Should `profile` be required or should we fallback to using `"default"`, which seems to be the behavior of other AWS tools?

Reply:
That's a good point. AFAIK, usually, most tools fallback to the `default` profile. I guess it would be better to make this property be optional.

Reply:
ef804b9